lib/index.js

```diff
@@ -13,7 +13,8 @@ var Utils = require('./utils');
 var internals = {
     initialized: false,
     defaults: {
-        logPath: Path.join(__dirname, '..', 'poop.log'),
+        logPath: Path.join(Path.dirname(process.mainModule.filename), 'poop.log'),
+        heapdumpFolder: Path.join(Path.dirname(process.mainModule.filename)),
```

Comment:
I'd prefer to use `process.cwd()` instead. It's simpler and works in the REPL (not that that is a huge deal).